e2e/testdata/hydration/deprecated-GK/kustomization.yaml

```diff
@@ -18,5 +18,6 @@ helmCharts:
   version: 0.2.0
   releaseName: my-prometheus-operator
 
-commonLabels:
-  test-case: hydration
+labels:
+- pairs:
```

Comment:
Should we set `includeSelectors: true` too? That will be consistent with the original behavior of `commonLabels`.

Reply:
I compared the output of the commands to what we are expecting and this is what matches right now.  I think the selectors was a noop in this case.  In case we change the payload and test we can potentially add more instructions.